_content/booklets/sixty-songs-of-milarepa_milarepa.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Sixty Songs of Milarepa" 
+authors:
+  - milarepa
+editor: "Bhikkhu Khantipālo"
+translator: "Garma C.C. Chang"
+external_url: "https://www.bps.lk/olib/wh/wh095_Chang_Sixty-Songs-of-Milarepa.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/18GKdHx22zXi0dpPHLUDqedP5Rr_B2b58/view?usp=sharing"
```

Comment:
epub as the second drive link: https://drive.google.com/file/d/1VkS2P_FJ1Q0FU5Bt1dNTGDgXqs16-r7q/view?usp=drivesdk